src/Components/HomePage/Homepage.js

```diff
@@ -1,14 +1,58 @@
 import React from 'react'
-import './Homepage'
+import './Homepage.scss'
+import { ActivityCards }  from '../ActivityCards/ActivityCards'
+import { CgUserlane } from "react-icons/cg";
+import { FaServer, FaGithub} from "react-icons/fa";
 
 export function Homepage() {
     return (
-        <main>
-            <article>HOLA</article>
-            <section>
-                <article>Adult Activities</article>
-                <article>Family Activities</article>
-            </section>
-        </main>
+        <div className='homepage-container'>
+
+          <nav className="nav-bar">
+            <div className="app-logo">
+              <CgUserlane className="logo"/>
+            </div>
+
+            <ul className="nav-links">
+              <li className="link">About</li>
+              <li className="link">Developers</li>
+              <li className="link"></li>
+            </ul>
+          </nav>
+
+          <div className="homepage-inner-container">
+            <h1 className="app-title1">Indecision</h1>
+            <h1 className="app-title2">maker</h1>
+            <p className="intro">Thus, our indecision may lie in our fears and doubts about the outcomes of our decisions or about the process of decision-making: </p>
+            <button className="find-activitty">find activity</button>
```

Comment:
As we discussed earlier today, I think this button on 27 should be changed to two buttons, one for `Find Family Activity` and another for `Find Adult Activity` to account for our wireframe and MVP.

Reply:
Will close this PR and will create the ticket for that. Thanks for the heads up.